client/client.go

```diff
@@ -550,18 +652,34 @@ type (
 		// Allows you to update the worker-build-id based version sets for a particular task queue. This is used in
 		// conjunction with workers who specify their build id and thus opt into the feature.
 		// WARNING: Worker versioning is currently experimental
+		// Deprecated: Use [UpdateWorkerVersioningRules] with the versioning-2 api.
 		UpdateWorkerBuildIdCompatibility(ctx context.Context, options *UpdateWorkerBuildIdCompatibilityOptions) error
 
 		// GetWorkerBuildIdCompatibility
 		// Returns the worker-build-id based version sets for a particular task queue.
 		// WARNING: Worker versioning is currently experimental
+		// Deprecated: Use [GetWorkerVersioningRules] with the versioning-2 api.
```

Comment:
We mention `versioning-2` a lot in this PR, is there a definition somewhere we can point users too?

Reply:
Good point. This is currently a placeholder, similar to 1.XX+ that you see in other places. When we had a new "formal" name and description of the new versioning API I will change that.